### ISSUE TITLE
work in progess // A11y Code Sprint Berlin: Dummy text / CONTENT

### ISSUE THREAD

**Issue by alexakressmann** [benjaminkott/bootstrap_package#1334]
**Pages**
-
https://bitv-v12.live.typo3.cps.321.works/
https://bitv-v12.live.typo3.cps.321.works/pages/simple
https://bitv-v12.live.typo3.cps.321.works/pages/feature
https://bitv-v12.live.typo3.cps.321.works/pages/startpage
https://bitv-v12.live.typo3.cps.321.works/pages/3-columns
https://bitv-v12.live.typo3.cps.321.works/pages/2-columns-offset-right
https://bitv-v12.live.typo3.cps.321.works/pages/subnavigation-left-and-2-columns
https://bitv-v12.live.typo3.cps.321.works/pages/2-columns-25/75
https://bitv-v12.live.typo3.cps.321.works/pages/subnavigation-right
https://bitv-v12.live.typo3.cps.321.works/pages/subnavigation-left
https://bitv-v12.live.typo3.cps.321.works/pages/default
https://bitv-v12.live.typo3.cps.321.works/pages/2-columns
https://bitv-v12.live.typo3.cps.321.works/pages/2-columns-50/50
https://bitv-v12.live.typo3.cps.321.works/pages/subnavigation-right-and-2-columns
https://bitv-v12.live.typo3.cps.321.works/content-examples/text/bullet-list
https://bitv-v12.live.typo3.cps.321.works/content-examples/text/text-and-icon
https://bitv-v12.live.typo3.cps.321.works/content-examples/text/list-group
https://bitv-v12.live.typo3.cps.321.works/content-examples/media/icon-group
https://bitv-v12.live.typo3.cps.321.works/content-examples/interactive/tab

https://bitv-v12.live.typo3.cps.321.works/content-examples/menus/abstract (Text in list)
https://bitv-v12.live.typo3.cps.321.works/content-examples/special-elements/insert-records (Text in table)

**Description**
-
All non-text content that is presented to the user has a text alternative that serves the equivalent purpose.
[To WCAG](https://www.w3.org/TR/WCAG21/#non-text-content)

**Solution**
-
Add dummy texts from this site space ipsum https://spaceipsum.com

**Comment by minapok** [comment]
On it.

**Comment by minapok** [comment]
Not on it anymore. Free for everyone to grab the ticket.

**Comment by RalleHo** [comment]
I take this!

**Comment by RalleHo** [comment]
I can't change the text in these two elements (boxed content & indented blocks). The text styles are not maintained and I cannot change them in the source view. Quick solution: both disabled.
![image](https://github.com/benjaminkott/bootstrap_package/assets/81409223/aba4db37-da7a-4974-b7ae-12a5c8c4c3b8)

**Comment by alexakressmann** [comment]
**Note 1: It is not possible to insert text on these pages**
https://bitv-v12.live.typo3.cps.321.works/pages/simple
https://bitv-v12.live.typo3.cps.321.works/pages/feature
https://bitv-v12.live.typo3.cps.321.works/pages/startpage
https://bitv-v12.live.typo3.cps.321.works/pages/3-columns
https://bitv-v12.live.typo3.cps.321.works/pages/2-columns-offset-right
https://bitv-v12.live.typo3.cps.321.works/pages/subnavigation-left-and-2-columns
https://bitv-v12.live.typo3.cps.321.works/pages/2-columns-25/75
https://bitv-v12.live.typo3.cps.321.works/pages/subnavigation-right
https://bitv-v12.live.typo3.cps.321.works/pages/subnavigation-left
https://bitv-v12.live.typo3.cps.321.works/pages/default
https://bitv-v12.live.typo3.cps.321.works/pages/2-columns
https://bitv-v12.live.typo3.cps.321.works/pages/2-columns-50/50
https://bitv-v12.live.typo3.cps.321.works/pages/subnavigation-right-and-2-columns

**Note 2: Remove bullet point in first list is not possible**
https://bitv-v12.live.typo3.cps.321.works/content-examples/menus/categorized

**Comment by alexakressmann** [comment]
QA successful
